cove_360/lib/threesixtygiving.py

```diff
@@ -396,6 +396,18 @@ def process(self, grant, grant_flat, path_prefix):
         self.message = "Users may find that the data is less useful as they are unable to discover more about the grants. Consider including a more detailed description if you have one."
 
 
+class TitleLength(AdditionalTest):
+    def process(self, grant, grant_flat, path_prefix):
+        title = grant.get("title", '')
```

Comment:
If there's no title field this test won't produce any output (`grant.get('title', '')` will avoid any `KeyError` raising a 500 in the server if there's no title field)